app/deploy/generator/06-syndesis-prometheus.yml.mustache

```diff
@@ -0,0 +1,165 @@
+- apiVersion: v1
+  kind: ServiceAccount
+  metadata:
+    name: syndesis-prometheus
+    labels:
+      app: syndesis
+      component: syndesis-prometheus
+
+- apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    labels:
+      app: syndesis
+      component: syndesis-prometheus
+    name: syndesis-prometheus-config
+  data:
+    prometheus.yml: |-
+      global:
+        scrape_interval:     5s
+        evaluation_interval: 5s
+
+      scrape_configs:
+        - job_name: prometheus
+          static_configs:
+            - targets:
+              - localhost:9090
+
+        - job_name: integration-pods
+
+          kubernetes_sd_configs:
+          - role: pod
+            namespaces:
+              names:
+                - ${OPENSHIFT_PROJECT}
+
+          relabel_configs:
+          - source_labels: [__meta_kubernetes_pod_annotation_prometheus_io_scrape]
+            action: keep
+            regex: true
```

Comment:
Not sure whether this is an issue for Prometheus, but using `true` instead of `"true"` for an otherwise string value (see the other regex) causes some trouble usually.

Reply:
It's using a different YAML parser and works fine. This is lifted from the [Prometheus example config](https://github.com/prometheus/prometheus/blob/master/documentation/examples/prometheus-kubernetes.yml#L150).